scaleway/table_scaleway_registry_image.go

```diff
@@ -0,0 +1,245 @@
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableScalewayRegistryImage(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:              "scaleway_registry_image",
+		Description:       "Registry Images allow you to manage your docker images in Scaleway.",
+		GetMatrixItemFunc: BuildRegionList,
+		List: &plugin.ListConfig{
+			Hydrate:       listRegistryImages,
+			ParentHydrate: listRegistryNamespaces,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "name",
+					Require: plugin.Optional,
+				}, {
+					Name:    "namespace_id",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getRegistryImage,
+			KeyColumns: plugin.AllColumns([]string{"id"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The Image name, unique in a namespace",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "namespace_id",
+				Description: "The unique ID of the Namespace the image belongs to",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("NamespaceID"),
+			},
+			{
+				Name:        "id",
+				Description: "A unique identifier of the instance.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "status",
+				Description: "The current status of the Image.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Status").Transform(transform.ToString),
+			},
+			{
+				Name:        "visibility",
+				Description: "A `public` image is pullable from internet without authentication, opposed to a `private` image. `inherit` will use the namespace `is_public` parameter.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Visibility").Transform(transform.ToString),
+			},
+			{
+				Name:        "size",
+				Description: "Image size in bytes, calculated from the size of image layers. One layer used in two tags of the same image is counted once but one layer used in two images is counted twice.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("Size").Transform(transform.ToInt),
+			},
+			{
+				Name:        "created_at",
+				Description: "The time when the Image was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "updated_at",
+				Description: "The time when the Image was updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "tags",
+				Description: "A list of tags attached with the docker image.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Scaleway standard columns
+			{
+				Name:        "project",
+				Description: "The ID of the project where the Image resides.",
```

Comment:
I don't think we need to use `Images`, using `image` in the column description should be fine.